services/core-web/src/components/mine/NoticeOfWork/MineNOWApplications.js

```diff
@@ -93,35 +87,33 @@ export class MineNOWApplications extends Component {
       ...persistedParams,
       // Overwrite prev params with any newly provided search params
       ...searchParams,
-      submissions_only: true,
     };
 
-    this.props.history.push(
+    this.props.history.replace(
       router.MINE_NOW_APPLICATIONS.dynamicRoute(this.props.mineGuid, updatedParams)
     );
   };
 
   render() {
     const isMajorMine = this.props.mines[this.props.mineGuid].major_mine_ind;
-    const title = isMajorMine ? "Permit Applications" : "Notice of Work Applications";
+    const type = isMajorMine ? "Permit Application" : "Notice of Work Application";
     return (
       <div className="tab__content">
         <div>
-          <h2>{title}</h2>
+          <h2>{type}s</h2>
```

Comment:
{types}

Reply:
That's intentional 😄 If you look at where else it is used

Reply:
haha wait whaaat, how is that correct syntax? 🤔